prime-dotnet-webapi-tests/Mocks/EnrolleeServiceMock.cs

```diff
@@ -146,24 +146,12 @@ public bool IsStatusChangeAllowed(Status startingStatus, Status endingStatus)
             return availableStatuses.Contains(endingStatus);
         }
 
-        public Task<bool> IsEnrolleeInStatusAsync(int enrolleeId, short statusCodeToCheck)
+        public Task<bool> IsEnrolleeInStatusAsync(int enrolleeId, params short[] statusCodesToCheck)
```

Comment:
Been trying to search why this is with no avail, but what does `params` do in the case of passing things into methods?